CMakeLists.txt

```diff
@@ -1089,17 +1089,69 @@ if (APPLE)
     execute_process(COMMAND chmod +x "${CMAKE_CURRENT_BINARY_DIR}/run-mixxx.sh")
   endif()
 else()
-    include(InstallRequiredSystemLibraries)
-    install(
-        FILES
-            ${CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS}
-        DESTINATION
-            "${MIXXX_INSTALL_BINDIR}"
-        COMPONENT
-            Libraries
+  include(InstallRequiredSystemLibraries)
+  install(
+    FILES
+        ${CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS}
+    DESTINATION
+        "${MIXXX_INSTALL_BINDIR}"
+    COMPONENT
+        Libraries
+  )
+endif()
+
+option(CODESIGN "Sign executables and libraries with digital certificate" OFF)
+mark_as_advanced(CODESIGN)
+if(CODESIGN)
+  set(SIGNTOOL_ARGS /a /t http://timestamp.verisign.com/scripts/timstamp.dll CACHE STRING "parameters for signtool (list)")
```

Comment:
```suggestion
  set(SIGNTOOL_ARGS /a /t http://timestamp.verisign.com/scripts/timestamp.dll CACHE STRING "parameters for signtool (list)")
```
typo?

Reply:
I don't think so, mysql uses the same: https://github.com/mysql/mysql-server/blob/67d52e7c7a1a23424e39273cbb6f5f9d56fda8d1/cmake/install_macros.cmake#L159

Do you know how signtool works? Using the `/a` flag apparently uses the best ceritficate available, but I have no idea how to make our cert available to signtool. If we could do that, we wouldn't need to specifiy custom codesign flags and wounldn't need to pass around env vars everywhere.

Reply:
I have no experience with `signtool`.